cli.js

```diff
@@ -38,7 +38,7 @@ if (config.jsdoc === false) {
 
 const cli = new CLIEngine({
   cwd: process.cwd(),
-  configFile: path.join(__dirname, 'eslintrc.json'),
+  baseConfig: require(path.join(__dirname, 'eslintrc.json')),
```

Comment:
The `baseConfig` option allows `.eslintrc` files to override it. I am `require`ing it here because it's expected to be an object, not a path.